lib/Runtime/Base/CharStringCache.cpp

```diff
@@ -18,15 +18,15 @@ namespace Js
     {
         AssertMsg(JavascriptString::IsASCII7BitChar(c), "GetStringForCharA must be called with ASCII 7bit chars only");
 
-        PropertyString * str = charStringCacheA[c];
+        PropertyString * str = charStringCacheA[(int)c];
```

Comment:
I don't like casting the index inside of the array index operator. I think, especially since you use this twice, you should assign the result of the cast to a new index variable and index with that variable instead.